apps/backoffice-v2/src/pages/Entity/components/Notes/Notes.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { ctw } from '@ballerine/ui';
+import { Link } from 'react-router-dom';
+import { Loader2, X } from 'lucide-react';
+
+import { Note } from './Note';
+import { NoteableType, TNotes } from './types';
+import { Form } from '@/common/components/organisms/Form/Form';
+import { Button } from '@/common/components/atoms/Button/Button';
+import { FormItem } from '@/common/components/organisms/Form/Form.Item';
+import { FormField } from '@/common/components/organisms/Form/Form.Field';
+import { Separator } from '@/common/components/atoms/Separator/Separator';
+import { FormControl } from '@/common/components/organisms/Form/Form.Control';
+import { FormMessage } from '@/common/components/organisms/Form/Form.Message';
+import { MinimalTiptapEditor } from '@/common/components/organisms/TextEditor';
+import { useNotesLogic } from '@/pages/Entity/components/Notes/hooks/useNotesLogic';
+import {
+  Sidebar,
+  SidebarContent,
+  SidebarGroup,
+  SidebarHeader,
+} from '@/common/components/organisms/Sidebar/Sidebar';
+
+const fallbackUser = {
+  id: 'test',
+  email: 'test',
+  phone: 'test',
+  lastName: 'test',
+  fullName: 'test',
+  avatarUrl: 'test',
+  createdAt: 'test',
+  firstName: 'test',
+  updatedAt: 'test',
+};
+
+export const Notes = ({
+  notes,
+  ...data
```

Comment:
We are passing className and future props directly to the form submit function and calling the props data we might want to reevaluate this.

Reply:
If and when we'll need that, we could take the `className` or any other prop out (the same way we did to `notes`) and deal with it then, it's not a generic UI component like `Button` that usually has `className` and similar props on it, so I don't think it should be done right now.